text/0314-cdk-construct-hub.md

```diff
@@ -0,0 +1,278 @@
+---
+rfc pr: [#314](https://github.com/aws/aws-cdk-rfcs/pull/314)
+tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/yyy
+---
+
+# CDK Construct Hub
+
+A community-driven hub for discovering and sharing CDK constructs.
+
+## Working Backwards
+
+### PRESS RELEASE
+
+Today, Amazon Web Services Inc., an Amazon company, announced the CDK Construct
+Hub, a service that makes it easy to find AWS Cloud Development Kit (CDK)
```

Comment:
General note: one of the goals of the construct hub is to be home to constructs for all CDK domains

Reply:
yes please this is already fractured

Reply:
I tried to rephrase most of it to "distance" from CDK (and related) to open up to "general" constructs as much as possible.